v2/goi18n/merge_command.go

```diff
@@ -125,12 +125,17 @@ func merge(messageFiles map[string][]byte, sourceLanguageTag language.Tag, outdi
 		}
 		templates := map[string]*i18n.MessageTemplate{}
 		for _, m := range mf.Messages {
-			templates[m.ID] = i18n.NewMessageTemplate(m)
+			template := i18n.NewMessageTemplate(m)
+			if template == nil {
+				fmt.Fprintf(os.Stderr, "missing translation for message id %q\n", m.ID)
```

Comment:
It isn't obvious to me that this should log an error. Can we just remove this?

Reply:
I disagree upon this point. A missing translation should not be ignored silently. The minimum I could agree on would be to change it to `os.Stdout`

Reply:
```suggestion
```